### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by rc**

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -135,6 +135,38 @@ Mesh-Related Tasks
...
+       region = domain.create_region(
+           'Region', 'vertices by get_region', 'facet',
+           functions=Functions([get_region]),
+       )
+
+   - Analogously a cell region defined using the coordinates of cell centroids::
+
+       # ...
+       region = domain.create_region(
+           'Region', 'cells by get_region', 'facet',
```
Comment (vlukes) [inline doc/faq.rst:184]: `facet` --> `cell`

File: doc/faq.rst [hunk 2]
```diff
@@ -67,6 +67,25 @@ Miscellaneous
...
 
+#. Visualization of various FEM-related information.
+
+   - Quadrature rules::
+
+       python3 script/plot_quadratures.py
+
+   - Facet orientations::
+
+       python3 sfepy/postprocess/plot_facets.py
```
Comment (vlukes) [inline doc/faq.rst:78]: Add::

```
import sys
sys.path.append('.')
```
to `plot_facets.py` script.

File: doc/faq.rst (hunk 1 above)
Comment (rc) [reply doc/faq.rst]: Done

File: doc/faq.rst (hunk 2 above)
Comment (rc) [reply doc/faq.rst:94]: This is a regular module, so I would rather not meddle with sys.path on its import. Instead, I explained PYTHONPATH setting in the FAQ.

Comment (vlukes) [reply doc/faq.rst:94]: OK

**Review by vlukes — APPROVED** [review]

**Pull request merged**